### DISCUSSION TITLE
Can't Deploy my Nextjs app

### DISCUSSION THREAD

**Discussion by Tabberplay** [vercel/community#682] [Help] [answered]
Hey,
when I want to Deploy my Nextjs app i get this error message: Module not found: `Can't resolve '/public/background.svg' in '/vercel/path0/pages'. `

How can i add the public folder to the paths so Vercel can build it?
In my Index.tsx I call my svg like this: `import indexContentPic from '../public/Background.svg'`

**Comment by swarnava** [comment, marked as answer]
If your build system is not copying/writing during build time, you can use `fs` method to write the file https://nodejs.org/api/fs.html#fs_fs_copyfile_src_dest_flags_callback
